vm-setup/roles/packages_installation/defaults/main.yml

```diff
@@ -1,21 +1,22 @@
 packages:
   ubuntu:
     packages:
-     - crudini 
-     - curl 
-     - dnsmasq 
-     - nmap 
-     - patch 
-     - psmisc 
+     - crudini
+     - curl
+     - dnsmasq
+     - ovmf
```

Comment:
This is added to the Ubuntu list, and `edk2-ovmf` below.  My editor removed the trailing whitespace, I can undo that if we want to keep it.